config/localization.yaml

```diff
@@ -1,9 +1,37 @@
+world_frame: "map"
+odom_frame: "odom"
+rover_frame: "base_link"
+use_odom_frame: false
+
 gps_linearization:
   reference_point_latitude: 42.2
   reference_point_longitude: -83.7
   reference_point_altitude: 224
+  publish_tf: false
+
+global_ekf:
+  # whether to use the Dilution of Precision (DoP) covariance estimate from the GPS message,
+  # or to replace it with the following configured covariance matrix
+  use_gps_dop_covariance: false
+  gps_covariance: [[2, 0, 0],
+                   [0, 2, 0],
+                   [0, 0, 2]]
+  
+  imu_orientation_covariance: [[0.1, 0, 0],
+                               [0, 0.1, 0],
+                               [0, 0, 0.1]]
+
+  imu_accel_covariance: [[0.1, 0, 0],
+                         [0, 0.1, 0],
+                         [0, 0, 0.1]]
+
+  imu_gyro_covariance: [[0.1, 0, 0],
```

Comment:
does not make sense to me that accel and gyro have the same values but different units

Reply:
Yeah the accel and gyro covariances were not actually tuned or anything, they're currently set to default values. I think this is fine for now since they aren't being integrated. The next part of this project will be measuring and tuning these on the rover and then integrating them to see if we can get better performance.